charts/etcd/templates/etcd-compaction-cronjob.yaml

```
@@ -41,7 +41,7 @@ spec:
 {{ toYaml .Values.annotations | indent 12 }}
 {{- end }}
           labels:
-            name: etcd
+            name: etcd-compaction-cronjob
```

```suggestion
            name: etcd-backup-compaction
```
Let's use the `name` label to indicate what process is actually running in the pod. Technically, it is not a cronjob, just a pod. Pedantic, I know, but still 😅